### DISCUSSION TITLE
2023 年终总结 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#20] [General]
# 2023 年终总结 | 高策

江湖小虾米

http://gaocegege.com/Blog/%E9%9A%8F%E7%AC%94/newyear2023

**Comment by yexia553** [comment]
第一次发现你，咱俩应该同龄，加油，希望你能如愿做出好的产品

**Comment by jiangzoi** [comment]
学长加油!

**Comment by whispererrr** [comment]
高老师未来还会更新博客吗？
